src/components/DevPanel.tsx

```diff
@@ -0,0 +1,369 @@
+      pdfType = PDFType.FORM;
+    } else if (userRole === Role.Client) {
+      pdfType = PDFType.IDENTIFICATION;
+    } else {
+      pdfType = undefined;
+    }
+
+    fetch(`${getServerURL()}/download`, {
+      method: 'POST',
+      credentials: 'include',
+      body: JSON.stringify({
+        fileId: documentId,
+        pdfType,
+      }),
+    }).then((response) => response.blob())
+      .then((response) => {
+        const pdfFile = new File([response], documentName, { type: 'application/pdf' });
+        console.log(pdfFile);
+
+        const url = window.URL.createObjectURL(response);
+        const a = document.createElement('a');
+        a.href = url;
+        a.download = documentName;
+        document.body.appendChild(a);
+        a.click();
+        a.remove();
+      }).catch((error) => {
+        alert('Error Fetching File');
+      });
+  }
+
+  ButtonFormatter = (cell, row, rowIndex, formatExtraData) => (
+    <div>
+      <label className="btn btn-filestack btn-widget ml-5 mr-5">
+        Re-Upload
+        <input type="file" accept="application/pdf" id="potentialPdf" multiple onChange={(event) => this.handleChangeFileUpload(event, rowIndex)} hidden />
+      </label>
+      <label className="btn btn-filestack btn-widget ml-5 mr-5">
+        Download
+        <button onClick={(event) => this.handleChangeFileDownload(event, rowIndex)} hidden />
+      </label>
+    </div>
+  )
+
+  tableCols = [{
+    dataField: 'filename',
+    text: 'Application Name',
+    sort: true,
+  }, {
+    dataField: 'category',
+    text: 'Category',
+    sort: true,
+  }, {
+    text: 'Actions',
+    formatter: this.ButtonFormatter,
+  }];
+
+  constructor(props: Props) {
+    super(props);
+    this.handleChangeFileUpload = this.handleChangeFileUpload.bind(this);
+    this.handleFileChange = this.handleFileChange.bind(this);
+    this.handleChangeFileDownload = this.handleChangeFileDownload.bind(this);
+    this.handleFileDownload = this.handleFileDownload.bind(this);
+    this.state = {
+      pdfFiles: undefined,
+      buttonState: '',
+      currentApplicationId: undefined,
+      currentApplicationFilename: undefined,
+      currentUser: undefined,
+      currentPage: 0,
+      itemsPerPageSelected: listOptions[0],
+      numElements: 0,
+      username: props.username,
+      searchName: '',
+      adminName: props.name,
+      organization: props.organization,
+      documents: [],
+    };
+    this.onClickWorker = this.onClickWorker.bind(this);
+    this.handleChangeSearchName = this.handleChangeSearchName.bind(this);
+    this.handleChangeItemsPerPage = this.handleChangeItemsPerPage.bind(this);
+    this.changeCurrentPage = this.changeCurrentPage.bind(this);
+    this.getDocuments = this.getDocuments.bind(this);
+    this.onChangeViewPermission = this.onChangeViewPermission.bind(this);
+    this.ButtonFormatter = this.ButtonFormatter.bind(this);
+  }
+
+  componentDidMount() {
```

Comment:
make all of these arrow functions

Reply:
The other code does not have arrow functions, we should change all the code at the same time

Reply:
If you are ok with that